### ISSUE TITLE
debugger doesn't show some mnemonics

### ISSUE THREAD

**Issue by aniou** [Trinity-11/FoenixIDE#6]
Hi,
looks like debugger doesn't show some mnemonics, see:

![obraz](https://user-images.githubusercontent.com/825100/61159741-54ecd880-a4fd-11e9-9d73-a9d1a75a226e.png)

corresponding .lst file:
```
.030074	a5 e9		lda $e9					lda MY_ROW
.030076	1a		inc a					inc a
.030077	1a		inc a					inc a
.030078	c5 13		cmp $13					cmp LINES_VISIBLE   	 ; we are at or over last line?
.03007a	b0 5e		bcs $0300da				bcs string_done    		 ; A >= LINES_VISIBLE
.03007c	85 e9		sta $e9					sta MY_ROW
```

**Comment by dtremblay** [comment]
This is probably due to spaces/tabs usage in editor.  I'll have to make this a little bit smarter.

**Comment by dtremblay** [comment]
Hi Aniou, could you try this again?  I've made a small fix in version 0.4.1.6 that should show your code now.

**Comment by dtremblay** [comment]
The way the source code is displayed was changed in version 0.4.1.6.
